test/integration/h2_capture_fuzz.proto

```diff
@@ -0,0 +1,158 @@
+syntax = "proto3";
+
+package test.integration;
+
+message H2FramePing {
+  enum Flags {
+    None = 0;
+    Ack = 1;
+  }
+  Flags flags = 1;
+  bytes data = 2;
+}
+
+message H2FrameSettings {
+  enum Flags {
+    None = 0;
+    Ack = 1;
+  }
+  Flags flags = 1;
+}
+
+enum HeadersFlags {
+  None = 0;
```

Comment:
Nit: for proto APIs, prefer all caps, e.g. NONE, etc.

Reply:
Done